src/rust/engine/process_execution/Cargo.toml

```diff
@@ -27,10 +27,13 @@ tokio-codec = "0.1"
 tokio-process = "0.2.1"
 tokio-timer = "0.2"
 workunit_store = { path = "../workunit_store" }
+tokio = "0.1"
 
 [dev-dependencies]
 maplit = "1.0.1"
 mock = { path = "../testutil/mock" }
 tempfile = "3"
 testutil = { path = "../testutil" }
+workunit_store = { path = "../workunit_store" }
```

Comment:
dependencies are implicitly dev-dependencies, so you shouldn't need workunit_store or tokio in here